coldfront/core/allocation/views.py

```diff
@@ -151,6 +150,7 @@ def get_context_data(self, **kwargs):
                     except AttributeError:
                         pass
                 allocation_user_su_usages[username] = usage
+                print(username, usage)
```

Comment:
Some `print`s in this file.

(Did you configure these to show up somewhere?)

Reply:
Yea, they get written to /etc/httpd/logs/cf_brc.error.log